File: pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/endpoints/channel_groups/AllChannelsChannelGroup.java
package com.pubnub.api.java.endpoints.channel_groups;

import com.pubnub.api.java.endpoints.Endpoint;
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAllChannelsResult;

public interface AllChannelsChannelGroup extends Endpoint<PNChannelGroupsAllChannelsResult> {
    AllChannelsChannelGroup channelGroup(String channelGroup);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/commonMain/kotlin/com/pubnub/kmp/endpoints/channel_groups/AllChannelsChannelGroup.kt
package com.pubnub.kmp.endpoints.channel_groups

import com.pubnub.kmp.PNFuture
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAllChannelsResult

/**
 * @see [PubNub.listChannelsForChannelGroup]
 */
interface AllChannelsChannelGroup : PNFuture<PNChannelGroupsAllChannelsResult>